packages/@guardian/eslint-plugin-source-foundations/package.json

```diff
@@ -28,14 +28,14 @@
 	"devDependencies": {
 		"@guardian/eslint-config": "^1.0.0",
 		"@guardian/eslint-config-typescript": "^1.0.0",
-		"@guardian/libs": "^3.0.0",
+		"@guardian/libs": "8.0.0",
 		"@types/estree": "^0.0.51",
 		"rollup": "^2.75.4",
 		"rollup-plugin-ts": "^2.0.7",
 		"typescript": "~4.6.4"
 	},
 	"peerDependencies": {
-		"@guardian/libs": "^3.0.0",
+		"@guardian/libs": "^8.0.5",
```

Comment:
did you mean to go with `8.0.5` here? 

the version we pin to in dev deps should be bottom of this range, so i think this should either be `"^8.0.5"` or devdeps pinned to `"8.0.0"`

Reply:
I think I did pin dev deps to "8.0.0", with peer deps being "^8.0.5", but in any case there's a new version of libs now so they're now all set to v9, with dev deps pinned and peer deps on "^". Is that what you'd expect?

Reply:
👍👍👍